metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorTableService.java

```diff
@@ -71,6 +71,7 @@
     private final HiveConnectorInfoConverter hiveMetacatConverters;
     private final HiveConnectorDatabaseService hiveConnectorDatabaseService;
     private final boolean allowRenameTable;
+    private final boolean onRenameConvertToExternal;
 
```

Comment:
Understand that having an option is good, but we have to maintain the option in the config or given a default value. Why don't we force the conversation during rename. I can't think of any reason to set this option as False. One case can be let the client to pass in the parameter to choose convert or not, but this option is global level, it can't do it.

Reply:
By default, the behavior is always not to move data. If a system wants to give the flexibility to the client, then they will set it to false. Clients will be able to move the data on move. If they don't want to, then they will first set it the table to external and then rename it.

Reply:
Sure, I guess we need to use the default as True always, given kragle can't make the change to V1 for setting the external parameter.